### ISSUE TITLE
Incorrect "inputs" in responses of getInputs and scriptInfo (Slot operator @ is not handled correctly)

### ISSUE THREAD

**Issue by MatthiasHinz** [duncantl/CodeDepends#4]
Hi,
I just tried to evaluate the following expression with getInputs (also with scriptInfo) and received an incorrect response for the inputs slot

`log(dataMALARIA@data$z * 10))`

The output for the slot is

```
Slot "inputs":
[1] "@"           "dataMALARIA" "data"  
```

I expected the value to be just `dataMALARIA`. From my testing I suppose that CodeDepends doesn't evaluate those expressions correctly that contain the @ operator. I tested with the package from the latest master-branch and R version 3.3.1, german locale.

Here is the complete sequence of commands:

```
> library(CodeDepends)
> getInputs(quote(log(dataMALARIA@data$z * 10)))
An object of class "ScriptNodeInfo"
Slot "files":
character(0)

Slot "strings":
character(0)

Slot "libraries":
character(0)

Slot "inputs":
[1] "@"           "dataMALARIA" "data"       

Slot "outputs":
character(0)

Slot "updates":
character(0)

Slot "functions":
log   * 
 NA  NA 

Slot "removes":
character(0)

Slot "nsevalVars":
character(0)

Slot "sideEffects":
character(0)

Slot "code":
log(dataMALARIA@data$z * 10)

```

**Comment by gmbecker** [comment]
I suspect this is because dataMALARIA@data is being interpreted as a normal function, rather than a data-accessor function, like `$` is. Should be an easy fix, I think, just need to use the `$` handler, or something very like it, for the `@` function.

I'll work something up but wait a bit for comment by @duncantl before committing.

Thanks for the report.

**Comment by gmbecker** [comment]
@MatthiasHinz  @duncantl I've addressed this in #5 

I'll leave it as a pull request for a day or two for comments, then merge it in. It turns out that while @ was (probably) not being handled correctly, that isn't actually where the problem was. It was actually the compound LHS that the dollar handler wasn't dealing with correctly.

i.e.

`dataMALARIA@data$z` is translated into the parser to `'$'(dataMALARIA@data, z)`, and then that first argument wasn't handled correctly in the dollar handler. That is now fixed for these types of recursive access-operator calls (`$` and `@`).

Please feel free to test it in my fork ( gmbecker/CodeDepends )

**Comment by gmbecker** [comment]
@MatthiasHinz Apologies for the large delay o this. Looks like I set up the pr right before going on paternity leave last year. 

Anyway, I've merged the PR that fixes this.  thanks for the report.
